stratum/hal/lib/tdi/es2k/es2k_sde_wrapper.cc

```diff
@@ -63,15 +83,25 @@ std::string GetBfChipId(int device) {
 
 }  // namespace
 
-std::string TdiSdeWrapper::GetChipType(int device) const {
+::util::StatusOr<bool> Es2kSdeWrapper::IsSoftwareModel(int device) {
```

Comment:
Technical debt. To the best of my knowledge, no one with specific knowledge of the ES2100 has ever gone back and cleaned up the original low-level ES2K target code (which was hastily adapted from the low-level Tofino code, IIRC).

Reply:
ok. Would recommend adding a `TODO` comment to address in future

Reply:
As a fervent advocate and long-time practitioner of the Boy Scout Rule, I would normally have changed this once I noticed it.

I chose not to do so in this case because the CL is already unwieldy. I do not want to distract the reviewer by changing anything that is not related to (a) implementing target-specific subclasses of `TdiSdeWrapper` and (b) eliminating the `TOFINO_TARGET` conditionals in the process.

I even undid a number of BSR changes I had automatically made (though I see I missed one).